### ISSUE TITLE
Confused by help msg for --analysis-mode 

### ISSUE THREAD

**Issue by tseemann** [cov-lineages/pangolin#396]
``` 
 --analysis-mode ANALYSIS_MODE
                        Specify which inference engine to use. Options:
                        accurate (UShER), fast (pangoLEARN), pangolearn,
                        usher. Default: UShER inference.
```
                        
The above text is confusing to me.
Are there 2 options or 3 options?
what are the option names? `usher` `pangoleanr` `accurate` `fast` ?                   
Or are some aliases for the others?

**Comment by AngieHinrichs** [comment]
The option names are `accurate`, `fast`, `pangolearn` and `usher`.  

`accurate` is an alias for `usher` (the default) and `fast` is an alias for `pangolearn`.

If that makes sense can you suggest a rewrite?  Thanks!

**Comment by aineniamh** [comment]
Yeah, this was to give two mode options: fast and accurate, which currently correspond to either pangolearn or usher. In future we might have more inference modes and fast/ accurate would correspond to the fastest or most accurate modes available then. 

It seemed important to also let the user be explicit about what inference method to use: usher, pangolearn ... whatever method might come along. 

Maybe the docs need to be clearer, or maybe this is more confusing than it needs to be. There's currently just 2 analysis options (that can be modified by skipping scorpio or designation hash mode), but they can be passed in either by specifying the software or the descriptor.

**Comment by tseemann** [comment]
Thanks @AngieHinrichs and @aineniamh 

Maybe something like this:

```
Pangolin supports two analysis approaches, USHER and PangoLearn.
USHER is the default and is selected using option "usher" or "accurate".
PangoLearn is selected using "pangolearn" or "fast".
```

**Comment by aineniamh** [comment]
Resolved in pangolin v4.0.3!
